src/metabase/models/spec_update.clj

```diff
@@ -0,0 +1,172 @@
+(ns metabase.models.spec-update
```

Comment:
A bit of a quibble, but what do you think about moving this to a `metabase.models.util.spec-update` namespace? I just worry that "top level" model namespaces look too much like actual models.